website/docs/sync.md

```diff
@@ -3,15 +3,38 @@ id: sync
 title: Replicating Data
 ---
 
-`Feature State`: The `Config` resource is currently alpha.
+## Replicating Data
 
-> The "Config" resource must be named `config` for it to be reconciled by Gatekeeper. Gatekeeper will ignore the resource if you do not name it `config`.
+Some constraints are impossible to write without access to more state than just the object under test. For example, it is impossible to know if an ingress's hostname is unique among all ingresses unless a ConstraintTemplate has access to all other ingresses. To enable this use case, we provide syncing of data into a data client.
 
-Some constraints are impossible to write without access to more state than just the object under test. For example, it is impossible to know if an ingress's hostname is unique among all ingresses unless a rule has access to all other ingresses. To make such rules possible, we enable syncing of data into OPA.
+### Replicating Data with SyncSets (preferred)
 
-The [audit](audit.md) feature does not require replication by default. However, when the ``audit-from-cache`` flag is set to true, the audit informer cache will be used as the source-of-truth for audit queries; thus, an object must first be cached before it can be audited for constraint violations.
+> `Feature State`: The `SyncSet` resource is currently in `alpha`.
 
-Kubernetes data can be replicated into the audit cache via the sync config resource. Currently resources defined in `syncOnly` will be synced into OPA. Updating `syncOnly` should dynamically update what objects are synced. Below is an example:
+Kubernetes data can be replicated into the data client using `SyncSet` resources. Below is an example of a `SyncSet`:
+
+```yaml
+apiVersion: syncset.gatekeeper.sh/v1alpha1
+kind: SyncSet
+metadata:
+  name: syncset-1
+  namespace: "gatekeeper-system"
+spec:
+  gvks:
+    - group: ""
+      version: "v1"
+      kind: "ConfigMap"
```

Comment:
WDYT about keeping all the example yamls consistent e.g. sync ingress like the scenario stated at the top?

Reply:
Happy to modify the SyncSet yaml example but the config yaml one links below to a demo file at https://raw.githubusercontent.com/open-policy-agent/gatekeeper/master/demo/basic/sync.yaml .

So modifying that snippet of yaml will probably require us modifying some part some part of the demo.

Alternatively, we can take out the "You can install this config with the following command:" bit. 

WDYT?

Reply:
or maybe update the scenario text to match the yamls?

Reply:
Oh right -- I updated the scenario.